keps/sig-storage/20200206-namespace-transfer.md

```diff
@@ -0,0 +1,217 @@
+---
+title: KEP Template
+authors:
+  - "@mhenriks"
+owning-sig: sig-storage
+participating-sigs:
+  - TBD
+reviewers:
+  - "@saad-ali"
+  - "@j-griffith"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2020-02-06
+last-updated: 2020-02-06
+status: provisional
+see-also:
+  - "/keps/sig-storage/20190709-csi-snapshot.md"
+  - "/keps/sig-storage/20181111-extend-datasource-field"
+replaces:
```

Comment:
I have no idea if it's appropriate to link https://github.com/kubernetes/enhancements/pull/1112 here?  Maybe that's to replace merged KEPs?  Regardless it does seem like some historical trail might be useful in here.

Reply:
I link to that in the "Implementation History" section but maybe it does make more sense here